elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -45,13 +59,27 @@ def render(element_html, data):
                 prefix = 'np.array('
                 suffix = ')'
 
+            # Mathematica reserved letters: C D E I K N O
+            mathematica_reserved = ['C', 'D', 'E', 'I', 'K', 'N', 'O']
+            if pl.inner_html(child) in mathematica_reserved:
+                mathematica_suffix = 'm'
+            else:
+                mathematica_suffix = ''
+
             # Create string for matlab and python format
-            matlab_data += pl.inner_html(child) + ' = ' + pl.string_from_2darray(var_data, language='matlab', digits=digits) + ';\n'
-            python_data += pl.inner_html(child) + ' = ' + prefix + pl.string_from_2darray(var_data, language='python', digits=digits) + suffix + '\n'
+            if commentExists == 0:
```

Comment:
I suggest replacing this with `if pl.has_attrib(child, 'params-comment'):` and deleting the `commentExists` variable.

(In case we keep `commentExists`, we should change it to be snake_case and use boolean True/False rather than 0/1)

Reply:
Doesn't make a difference to me, removed `commentExists` variable with suggested line